packages/editor/styles.css

```
@@ -127,6 +127,89 @@
+  display: none;
+}
+
+.tuturuuu-editor
+  .tiptap
+  summary:not(:has(> .tuturuuu-editor-collapsible-toggle))::before {
+  display: block;
+  width: 0;
+  height: 0;
+  flex: 0 0 auto;
+  border-block: 0.3rem solid transparent;
+  border-inline-start: 0.45rem solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)